gradle.properties

```diff
@@ -2,7 +2,7 @@ projectVersion=2.1.1.BUILD-SNAPSHOT
 micronautBuildVersion=1.1.5
 jacksonVersion=2.11.2
 micronautDocsVersion=1.0.24
-micronautVersion=2.0.1
+micronautVersion=2.1.2
```

Comment:
Unless this is really necessary (which I don't think it is), it's better to keep this module in Micronaut 2.0. This way users that don't want to upgrade can still use it (because everything happens at compile time).

Reply:
Thanks, @ilopmar. Accordingly, I narrowed down the version bump to the bare minimum.